tasks/__init__.py

```diff
@@ -537,6 +537,11 @@ def test_js(ctx):
     jshint(ctx)
     karma(ctx, single=True, browsers='PhantomJS')
 
+@task
+def test_travis_js(ctx, numprocesses=None):
```

Comment:
Since we're putting this in api1, I don't think we need this task for travis. Also, could you rename the branch on travis so it's called `api1_and_js` or something?